lib/Echidna/Transaction.hs

```diff
@@ -131,7 +131,9 @@ canShrinkTx _                                = True
 shrinkTx :: MonadRandom m => Tx -> m Tx
 shrinkTx tx'@(Tx c _ _ _ gp (C _ v) (C _ t, C _ b)) = let
   c' = either (fmap Left . shrinkAbiCall) (fmap Right . pure) c
+  lower 0 = pure $ w256 0
   lower x = w256 . fromIntegral <$> getRandomR (0 :: Integer, fromIntegral x)
+              >>= \r -> uniform [0, r] -- try 0 quicker
```

Comment:
maybe we want to be explicit about `uniform` wherever we use `lower`. pinging @japesinator for this one.

Reply:
I've just put it here because all calls to lower should use this strategy, it is a local function.